pykafka/producer.py

```diff
@@ -17,17 +17,25 @@
 See the License for the specific language governing permissions and
 limitations under the License.
 """
-__all__ = ["Producer", "AsyncProducer"]
+__all__ = ["Producer", "SynchronousProducer"]
```

Comment:
Was going to comment about this being a breaking change, but it looks like this is how the JVM client is named too. However, I'd recommend `SyncProducer`. If Java has the short name, let's not out-length it.